app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="page-header">
+  <%= @organization.login %>
+</h1>
+
+<%= link_to 'Remove Classroom', @organization, class: 'btn btn-sm btn-danger', role: 'button',
```

Comment:
Another thing to think about, what happens when a user that's associated with a classroom is removed from the owners team? This is another place where you might need to worry about synchronization or getting clever about how you model things.